.rubocop_todo.yml

```diff
@@ -0,0 +1,53 @@
+# This configuration was generated by `rubocop --auto-gen-config`
+# on 2015-07-16 18:12:35 +0530 using RuboCop version 0.31.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: AllowSafeAssignment.
+Lint/AssignmentInCondition:
+  Enabled: false
+
+# Offense count: 14
+Metrics/AbcSize:
+  Max: 33
+
+# Offense count: 1
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 133
+
+# Offense count: 1
+Metrics/CyclomaticComplexity:
+  Max: 12
+
+# Offense count: 108
+# Configuration parameters: AllowURI, URISchemes.
+Metrics/LineLength:
+  Max: 148
+
```

Comment:
You need to break them, align them in such a way that they don't get wider than 100 characters. We need to customize our code that it passes rubocop and not customize rubocop that our code passes.

I think I have mentioned this [ruby-style-guide](https://github.com/bbatsov/ruby-style-guide). Check out the section where they talk about line length. You can also have a look at these:
* http://stackoverflow.com/questions/10522414/breaking-up-long-strings-on-multiple-lines-in-ruby-without-stripping-newlines
* http://stackoverflow.com/questions/12083011/how-to-break-up-long-lines-of-ruby
* http://codereview.stackexchange.com/questions/36782/how-do-i-refactor-lines-of-ruby-code-that-run-too-long-due-to-method-chaining-or

If you come across something which you can't your head around then you should let me know.